lib/blacklight/maps/maps_search_builder.rb

```diff
@@ -0,0 +1,20 @@
+module BlacklightMaps
+  module MapsSearchBuilder
```

Comment:
Yep, and you specify the search building in CatalogController, like this: https://github.com/geoblacklight/geoblacklight/blob/master/lib/generators/geoblacklight/templates/catalog_controller.rb#L24